themes/default/content/blog/announcing-6-0-of-the-pulumi-aws-classic-provider/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing 6.0 of the Pulumi AWS Classic Provider"
+date: 2023-08-29T09:00:00-04:00
+draft: false
+meta_desc: "Announcing 6.0 of the Pulumi AWS Classic Provider, including SDK size optimization, TF plugin framework support, new resources, and more."
+meta_image: meta.png
+authors:
+    - mikhail-shilkov
+    - monica-rodriguez
+tags:
+    - aws
+    - providers
+
+---
+
+We are excited to announce 6.0 of the [Pulumi AWS Classic provider](https://www.pulumi.com/registry/packages/aws/). The AWS Classic provider is the most heavily used provider across the entire Pulumi ecosystem, and offers access to the full surface area of the upstream Terraform AWS Provider from within Pulumi projects in all supported Pulumi languages. The [6.0 release](https://github.com/pulumi/pulumi-aws/releases/tag/v6.0.2) brings a substantial set of fixes and improvements to the provider, including a number of breaking changes as part of the major version release.
```

Comment:
I don't love the link to the GH release tag because it looks like nothing changed:

<img width="1273" alt="image" src="https://github.com/pulumi/pulumi-hugo/assets/410614/e3624bd9-0ad1-49a0-90f6-ec48f42ef189">

I don't know what would look better.

Reply:
Thank you. I've pinged the team to see if we can add some more details here. cc @mikhailshilkov